.modernizrrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "minify": true,
+  "options": [
+    "setClasses"
+  ],
+  "feature-detects": [
+    "test/webgl",
+    "test/css/flexbox"
```

Comment:
Note: at present we are not using flexbox, I dropped it from the header work. Unless @Chrissy is using it for the map work, I think we can drop that here safely.